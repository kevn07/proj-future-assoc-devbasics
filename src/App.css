.App {
  text-align: center;
}
.StyledButton {
  width: 365px;
  border-radius: 10px;
  padding: 1rem;
  background-color: #2ea44f;
  border: none;
  color: white;
  font-size: 16px;
  font-weight: 700;
}

.StyledButton:hover,
.StyledButton:active {
  background-color: #2c974b;
  cursor: pointer;
}
.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
