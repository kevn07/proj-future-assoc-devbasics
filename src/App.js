import logo from "./logo.svg";
import "./App.css";
import { MichaelNguyen } from "./components/MichaelNguyen/MichaelNguyen";
import { EthanMoliver } from "./components/EthanMoliver/EthanMoliver";
import { DanielChegwidden } from './components/DanielChegwidden/DanielChegwidden';


function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>Name | Mantel Brand | Favourite Food | Last show watched</p>
        <MichaelNguyen />
        <EthanMoliver />
        <DanielChegwidden /> 

      </header>
    </div>
  );
}

export default App;
