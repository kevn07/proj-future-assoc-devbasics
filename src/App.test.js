import { render, screen } from "@testing-library/react";
import App from "./App";
import renderer from "react-test-renderer";

test("heading should always be there", () => {
  render(<App />);
  const heading = screen.getByText(/Fetch a Pokemon/i);
  expect(heading).toBeInTheDocument();
});

test("page renders correctly", () => {
  render(<App />);
  const page = renderer.create(<App />).toJSON();
  expect(page).toMatchSnapshot();
});
