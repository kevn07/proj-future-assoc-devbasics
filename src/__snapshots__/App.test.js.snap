// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
<<<<<<< HEAD
    <h3>
      Fetch a Pokemon
    </h3>
=======
    <img
      alt="pokemon-logo"
      src="pokemon_logo.svg"
      width="500px"
    />
    <br />
>>>>>>> main
    <br />
    <button
      className="StyledButton"
      onClick={[Function]}
    >
      Fetch Pokemon
    </button>
  </header>
</div>
`;
